cx34/cx34_registers.go

```diff
@@ -251,6 +256,7 @@ Table of registers with values that changed
 
 // Known Register values.
 const (
+	SwitchOnOff                Register = 140 // 0 = off, 1 = on
```

Comment:
I can't be sure since I don't have my unit yet but that's what the documentation linked in the PR description seems to suggest. I assume it's the equivalent of putting the unit into "standby" mode via the hardwired heat/cool/standby terminals. Please tell me if you discover something different!

Reply:
@sodabrew or @metcalf - did either of you try this out? I haven't checked this PR out locally yet to try it.

Reply:
I don't have my unit yet so I haven't been able to test it.